recipes-app/node-red-gpio/files/0003-mraa-gpio-din-Make-D14-D19-IOT2050-only.patch

```diff
@@ -0,0 +1,41 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Baocheng Su <baocheng.su@siemens.com>
+Date: Thu, 28 Dec 2023 09:56:02 +0800
+Subject: [PATCH] mraa-gpio: din: Make D14-D19 IOT2050 only
+
+Pins D14-D19 are IOT2050 only.
+
+Fixes: 25d7d668ef11 ("Add IOT2050 board into mraa Intel nodes (#940)")
+Signed-off-by: Baocheng Su <baocheng.su@siemens.com>
+---
+ hardware/intel/mraa-gpio-din.html | 10 ++++------
+ 1 file changed, 4 insertions(+), 6 deletions(-)
+
+diff --git a/hardware/intel/mraa-gpio-din.html b/hardware/intel/mraa-gpio-din.html
+index c7fb92e0e9b4..1fa3c443ee11 100644
+--- a/hardware/intel/mraa-gpio-din.html
++++ b/hardware/intel/mraa-gpio-din.html
+@@ -22,6 +22,10 @@
+         },
+         oneditprepare: function() {
+             if (RED.settings.mraaGpioDinBoardType === "IOT2050") {
++                for (i of [...Array(20).keys()].slice(14)) {
```

Comment:
why not `for (let i = 14; i <= 19; i++)`?

Reply:
Good idea, my javascript sucks :)